src/chrome/content/rules/GoNewSoft.com.xml

```diff
@@ -1,7 +0,0 @@
-<ruleset name="GoNewSoft.com">
-        <target host="gonewsoft.com" />
-        <target host="www.gonewsoft.com" />
```

Comment:
So then you need to revert 1b127c4, otherwise I can not merge your branch without removing it.